openlibrary/macros/CreateList.html

```diff
@@ -0,0 +1,41 @@
+<div class="widget-add">
```

Comment:
Have you tested this locally? I believe you will need this at the top of the file: `$def with (...)`

Reply:
I put this div in the sidebar rather than CreateList